packages/manager/cypress/e2e/core/linodes/migrate-linode.spec.ts

```diff
@@ -0,0 +1,253 @@
+    mockAppendFeatureFlags({
+      dcSpecificPricing: makeFeatureFlagData(true),
+    }).as('getFeatureFlags');
+    mockGetFeatureFlagClientstream().as('getClientStream');
+
+    // Mock request to get Linode volumes and disks
+    mockGetLinodeDisks(mockLinode.id, mockLinodeDisks).as('getLinodeDisks');
+    mockGetLinodeVolumes(mockLinode.id, []).as('getLinodeVolumes');
+
+    // Mock requests to get all Linode types, and to get individual types.
+    mockGetLinodeType(dcPricingMockLinodeTypes[0]).as('getLinodeTypeOld');
+    mockGetLinodeType(dcPricingMockLinodeTypes[1]).as('getLInodeTypeNew');
+
+    cy.visitWithLogin(`/linodes/${mockLinode.id}?migrate=true`);
+    cy.wait([
+      '@getClientStream',
+      '@getFeatureFlags',
+      '@getLinode',
+      '@getLinodeDisks',
+      '@getLinodeVolumes',
+    ]);
+
+    ui.button.findByTitle('Enter Migration Queue').should('be.disabled');
+    cy.findByText(`${initialRegion.label}`).should('be.visible');
+    getClick('[data-qa-checked="false"]');
+    cy.findByText(`North America: ${initialRegion.label}`).should('be.visible');
+    containsClick(selectRegionString);
+
+    ui.regionSelect.findItemByRegionLabel(newRegion.label).click();
+
+    cy.findByText(dcPricingCurrentPriceLabel).should('be.visible');
+    const currentPrice = dcPricingMockLinodeTypes[0].region_prices.find(
+      (regionPrice) => regionPrice.id === initialRegion.id
+    );
+    cy.get('[data-testid="current-price-panel"]').within(() => {
+      cy.findByText(`$${currentPrice.monthly.toFixed(2)}`).should('be.visible');
+      cy.findByText(`$${currentPrice.hourly}`).should('be.visible');
+      cy.findByText(
+        `$${dcPricingMockLinodeTypes[0].addons.backups.price.monthly.toFixed(
+          2
+        )}`
+      ).should('be.visible');
+    });
+
+    cy.findByText(dcPricingNewPriceLabel).should('be.visible');
+    const newPrice = dcPricingMockLinodeTypes[1].region_prices.find(
+      (linodeType) => linodeType.id === newRegion.id
+    );
+    cy.get('[data-testid="new-price-panel"]').within(() => {
+      cy.findByText(`$${newPrice.monthly.toFixed(2)}`).should('be.visible');
+      cy.findByText(`$${newPrice.hourly}`).should('be.visible');
+      cy.findByText(
+        `$${dcPricingMockLinodeTypes[1].addons.backups.price.monthly.toFixed(
+          2
+        )}`
+      ).should('be.visible');
+    });
+
+    // intercept migration request and stub it, respond with 200
+    cy.intercept(
+      'POST',
+      apiMatcher(`linode/instances/${mockLinode.id}/migrate`),
+      {
+        statusCode: 200,
+      }
+    ).as('migrateReq');
+
+    fbtClick('Enter Migration Queue');
+    cy.wait('@migrateReq').its('response.statusCode').should('eq', 200);
+  });
+
+  /*
+   * - Confirms DC-specific pricing UI flow works as expected during Linode migration.
+   * - Confirms that pricing comparison is not shown in "Region" section migration occurs in the same DC.
```

Comment:
```suggestion
   * - Confirms that pricing comparison is not shown in "Region" section if migration occurs in a DC with the same price structure.
```

Here, it is the same DC, because that's the mock data we have to work with, but for clarity, it's probably best to note that the price notice shouldn't show up if the region prices are the same -- even if they're different regions.